README.md

```diff
@@ -113,6 +113,95 @@ The test cases execution depends on the device capability, so it's
 important to fill in the device capabilities section correctly in the test
 configuration file. We'll discuss this in more detail in the following sections.
 
+The DBS capable device pair with both `enable_sta_dfs_channel_for_peer_network`
```

Comment:
What is the purpose of outlining test cases for this specific set of device capabilities? Is it a default configuration?